engine/class_modules/priest/sc_priest_shadow.cpp

```diff
@@ -1133,27 +1133,24 @@ struct void_bolt_t final : public priest_spell_t
 
       td.dots.shadow_word_pain->adjust_duration( dot_extension, true );
       td.dots.vampiric_touch->adjust_duration( dot_extension, true );
-
-      if ( priest().conduits.dissonant_echoes->ok() && priest().buffs.voidform->check() )
-      {
-        if ( rng().roll( priest().conduits.dissonant_echoes.percent() ) )
-        {
-          priest().cooldowns.void_bolt->reset( true );
-          priest().procs.dissonant_echoes->occur();
-        }
-      }
     }
   };
 
   void_bolt_extension_t* void_bolt_extension;
   propagate_const<cooldown_t*> shadowfiend_cooldown;
   propagate_const<cooldown_t*> mindbender_cooldown;
+  timespan_t hungering_void_base_seconds;
```

Comment:
Just name the with _duration instead of seconds. It is a timespan type so you don't care what unit someone used to initialize its value.